pylutron/__init__.py

```diff
@@ -926,7 +924,7 @@ def handle_update(self, args):
     action = int(args[0])
     if action != OccupancyGroup._ACTION_STATE or len(args) != 2:
       return False
-    self._state = int(args[1])
+    self._state = OccupancyGroup.State(int(args[1]))
```

Comment:
what happens when we get an invalid value? Is there a way to check if the value is a valid enum, and only set it then, otherwise force it to UNKNOWN yourself?